src/hor2eq.jl

```diff
@@ -0,0 +1,137 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _hor2eq(alt::T, az::T, jd::T, ws::Bool, B1950::Bool,
+                 precession::Bool, nutate::Bool, aberration::Bool,
+                 refract::Bool, lat::T, lon::T, altitude::T, pressure::T,
+                 temperature::T, obsname::AbstractString) where {T<:AbstractFloat}
+
+    if obsname == ""
+        # Using Pine Bluff Observatory values
+        if isnan(lat)
+            lat = T(43.0783)
+        end
+
+        if isnan(lon)
+            lon = T(-89.865)
+        end
+    else
+        lat = observatories[obsname].latitude
+        lon = observatories[obsname].longitude
+        altitude = observatories[obsname].altitude
+    end
+
+    if refract
+        alt_b = co_refract(alt, altitude, pressure, temperature)
+    else
+        alt_b = alt
+    end
+
+    if ws
+        az -= 180
+    end
+    dra1, ddec1, eps, d_psi, _ = co_nutate(jd, 45, 45)
```

Comment:
`_` is useful to discard leading or intermediate elements of an iterable (`_, b, _, d = 1, 2, 3, 4`), but for trailing elements there is no need to use `_`,
```julia
_, _, eps, d_psi = co_nutate(jd, 45, 45)
```
should be sufficient ;-)